server/plugin.go

```diff
@@ -413,6 +414,10 @@ func (p *Plugin) GetSiteURL() string {
 	return p.getConfig().mattermostSiteURL
 }
 
+func (p *Plugin) CreateFullURLPath(extensionPath string) string {
```

Comment:
I know the code for this is trivial, but It might change in the future, so it would be nice having a test to control whether we break it in the future with new logic. :-)

Let me send you a test example for this!